tools/triage/add_triage_signature.py

```diff
@@ -1103,6 +1103,81 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket(report)
 
 
+class NameserverInClusterNetwork(ErrorSignature):
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            function_impact_label="bad_nameserver",
+            comment_identifying_string="h1. Nameserver in internal network",
+        )
+
+    def _get_resolv_conf_files(self, url, base_path):
+        files = []
+        try:
+            triage_logs_tar = get_triage_logs_tar(triage_url=url, cluster_id=get_metadata_json(url)["cluster"]["id"])
+        except FileNotFoundError:
```

Comment:
iiuc you shouldn't catch here the exception
otherwise how can you fallback to the older format at line 1172

Reply:
I thought that the old/new format are part of the base_path while here we do not have logs at all. this is why I want to separate the cases. WDYT?

Reply:
ack. then it's ok as-is
thanks!